tests/unit/Core/Framework/Adapter/Cache/Http/CacheStoreTest.php

```diff
@@ -207,7 +207,13 @@ public function testLookupWithSoftPurgeAndStaleCache(): void
             $maintenanceResolver->expects($this->once())->method('shouldBeCached')->willReturn(true);
 
             $kernel = $this->createMock(HttpKernelInterface::class);
-            $kernel->expects($this->once())->method('handle')->willReturn($response);
+
+            $originalRequest = $request;
+            $kernel->expects($this->once())->method('handle')->willReturnCallback(function (Request $request) use($response, $originalRequest) {
+                static::assertNotSame($request, $originalRequest);
```

Comment:
for comparing objects, we should use 
```suggestion
                static::assertNotEquals($request, $originalRequest);
```

as notSame is almost true in every case. because even the object reference need to be same. but here we also want to check, if the session is different, right?

Reply:
symfony declines too merge my PR. so I need to rewrite anything aaaaa.